lib/dealership.rb

```diff
@@ -0,0 +1,65 @@
+class Dealership
+  attr_reader :name, :address, :inventory
+  def initialize(name, address)
+    @name = name
+    @address = address
+    @inventory = []
+  end
+
+  def inventory_count
+    @inventory.length
+  end
+
+  def has_inventory?
+    if @inventory == []
+      false
+    else
+      true
+    end
+  end
+
+  def add_car(car)
+    @inventory << car
+  end
+
+  def cars_by_make(make)
+    same_make = []
+    inventory.each do |car|
+      if car.make == make
+        same_make << car
+      end
+    end
+    return same_make
+  end
+
+  def total_value
+    value = 0
+    inventory.each do |car|
+      value += car.total_cost
+    end
+    return value
```

Comment:
```suggestion
    value = 0
    inventory.each do |car|
      value += car.total_cost
    end
   value
```

Ruby methods implicitly return whatever the last line of the method is so we do not need to explicitly say `return`